docs/ref/django-admin.txt

```diff
@@ -1651,6 +1651,11 @@ the deletion.
 
 Specifies the database to use. Defaults to ``default``.
 
+.. django-admin-option:: --include-stale-apps
+
+Deletes stale content types including ones whose app isn't installed or have
+been deleted. Defaults to ``False``.
+
```

Comment:
I'm guessing there needs to be a `.. versionadded:: 3.1` somewhere here, but not sure what it should look like.

Reply:
```
.. django-admin-option:: --include-stale-apps 

.. versionadded:: 3.1
```